src/niscope/metadata/functions_addon.py

```diff
@@ -384,6 +387,125 @@
 digitizer acquires for the specified channel. This function returns
 scaled voltage waveforms.
 
+This function may return multiple waveforms depending on the number of
+channels, the acquisition type, and the number of records you specify.''',
+            'note': 'Some functionality, such as time stamping, is not supported in all digitizers.',
+        },
+    },
+    'FancyRead': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'The instrument handle you obtain from niScope_init that identifies a particular instrument session.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'channelList',
+                'type': 'ViChar[]',
+                'documentation': {
+                    'description': 'The channel to configure.',
```

Comment:
Not that it matters to the public aspects of the project, but in this case we don't actually configure the channel.

Reply:
Fixed.